storybook/src/docs/bestPractices/GridAndFormLayouts.stories.tsx

```diff
@@ -975,7 +932,7 @@ export const GridWithSelectionInDialog = {
                     <ToolbarAutomaticTitleItem />
                     <ToolbarFillSpace />
                     <ToolbarActions>
-                        <Button color="primary" variant="contained" startIcon={<SelectIcon />} onClick={() => setShowDialog(true)}>
+                        <Button color="primary" variant="contained" startIcon={<SelectIcon />} onClick={() => editDialogApi.openAddDialog()}>
```

Comment:
This should probably be openEditDialog, since you're editing the current selection.

Reply:
I cannot use .openEditDialog here, because it expects an id argument. As this dialog does not edit a specific element, I can't use that here.